packages/http-client-csharp/generator/TestProjects/CadlRanch/http/payload/media-type/tspCodeModel.json

```diff
@@ -207,7 +207,7 @@
         "CrossLanguageDefinitionId": "TypeSpec.string",
         "Decorators": []
        },
-       "BodyMediaType": "Json",
+       "BodyMediaType": "Text",
```

Comment:
return a pure string, so the media type is `Text`. Note that this happens to be correct. Deducing media type from return data type is not accurate.